File: phaseGateOne/taskThree.py
firstNumber = int(input("Enter a number:"))
secondNumber = int(input("Enter a number:"))
thirdNumber = int(input("Enter a number:"))

if firstNumber > secondNumber or firstNumber > thirdNumber:
	print(firstNumber)
if secondNumber > firstNumber or secondNumber > thirdNumber:
	print(secondNumber )
if thirdNumber> firstNumber or thirdNumber > secondNumber:
	print(thirdNumber)

File: phaseGateOne/TaskThree.java

import java.util.Scanner;
public class TaskThree{
public static void main(String[] args){
 Scanner scanner = new Scanner(System.in);

System.out.print("Enter a number:");
int firstNumber = scanner.nextInt();

System.out.print("Enter a number:");
int secondNumber = scanner.nextInt();

System.out.print("Enter a number:");
int thirdNumber = scanner.nextInt();


if(firstNumber > secondNumber || firstNumber > thirdNumber ){
 System.out.println(firstNumber);
}
if(secondNumber > firstNumber || secondNumber > thirdNumber ){
 System.out.println(secondNumber);
}
if(thirdNumber> firstNumber || thirdNumber > secondNumber ){
 System.out.println(thirdNumber);
}

}
}
